src/modules/fancyzones/lib/JsonHelpers.cpp

```diff
@@ -321,6 +321,40 @@ namespace JSONHelpers
         }
     }
 
+    void FancyZonesData::UpdatePrimaryDesktopData(const std::wstring& desktopId)
```

Comment:
Add a description to explain what this function does. When a function has to workaround a Windows bug, the code it's not self explanatory.

Reply:
Updated description.